CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Digital Marketplace Visual Regression changelog
+
+Records breaking changes from major version bumps
+
+## 2.0.0
+
+PR: [#152](https://github.com/alphagov/digitalmarketplace-apiclient/pull/152)
```

Comment:
Er... is this the right changelog entry?

Reply:
bum. No, I coped the changelog from the apiclient and forgot to update it.. will fix

Reply:
Cool, makes sense.